lib/client/components/Location.jsx

```diff
@@ -11,17 +12,18 @@ export default class Location extends React.Component {
   }
 
   handleClick() {
-    navigator.geolocation.getCurrentPosition(
+    this.setState({ location: 'Fetching location...', error: '' });
+
+    Geolocation.getCurrentPosition(
       geo => {
         const lat = geo.coords.latitude;
         const lon = geo.coords.longitude;
 
-        this.setState({ lat, lon, location: `${lat}, ${lon}`, error: '' });
+        this.setState({ lat, lon, location: `${lat.toFixed(4)}, ${lon.toFixed(4)}`, error: '' });
```

Comment:
Is there some benefit to rounding off the coordinates this way? Doesn't it lose precision?